sam.c

```diff
@@ -449,7 +450,7 @@ void bam_cigar2rqlens(int n_cigar, const uint32_t *cigar, int *rlen, int *qlen)
     }
 }
 
-int bam_cigar2qlen(int n_cigar, const uint32_t *cigar)
+hts_pos_t bam_cigar2qlen(int n_cigar, const uint32_t *cigar)
```

Comment:
If changing the return type, the type of `l` needs to change too.

Reply:
This shows the danger of updating code while jet-lagged.  Should now be fixed by 8ac3b5c4.